include/NAS2D/StringUtils.h

```diff
@@ -17,6 +17,44 @@
 
 namespace NAS2D
 {
+	template<typename T>
+	T from_string(const std::string& value)
+	{
+		return T(value);
+	}

```

Comment:
This method could probably use an `std::enable_if` guard, to make sure it is only considered when `T` is constructible from a `std::string`. I'd have to think about the exact syntax needed to do that. Maybe a tomorrow thing.

Reply:
I don't have much (re: any) experience with using `std::enable_if`, definitely a tomorrow thing.

Reply:
At least it's documented that construction from `std::string` is required.